vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -6,6 +6,18 @@
 
 using Method = facebook::xplat::module::CxxModule::Method;
 
+#ifndef RNW_PKG_VERSION_MAJOR
+#define RNW_PKG_VERSION_MAJOR 1000
```

Comment:
Why 1000? Could we stick with 0.0.0 for development builds to align with RN version (and likely actual)?

Reply:
Same in the resource file